adapter/config/parser.go

```diff
@@ -34,15 +34,23 @@ import (
 )
 
 var (
-	onceConfigRead    sync.Once
-	onceLogConfigRead sync.Once
-	onceGetMgwHome    sync.Once
-	adapterConfig     *Config
-	adapterLogConfig  *LogConfig
-	mgwHome           string
-	e                 error
+	onceConfigRead      sync.Once
+	onceGetDefaultVhost sync.Once
+	onceLogConfigRead   sync.Once
+	onceGetMgwHome      sync.Once
+	adapterConfig       *Config
+	defaultVhost        map[string]string
+	adapterLogConfig    *LogConfig
+	mgwHome             string
+	e                   error
```

Comment:
shall we use error instead of e?

Reply:
This was already in the code, just added `onceLogConfigRead` and `defaultVhost`, shall we refactor this with another PR? the indentation is the change